web/src/main/java/org/cbioportal/web/StudyViewController.java

```diff
@@ -453,6 +489,13 @@ private void init() {
                 densityPlotBin.setMaxY(yValueBigDecimal);
             }
         }
+        
+        result.setPearsonCorr(
+            new PearsonsCorrelation().correlation(xValues, yValues)
+        ); 
+        result.setSpearmanCorr(
+            new SpearmansCorrelation().correlation(xValues, yValues)
+        );
```

Comment:
What do you have your column count set to? On the backend, we don't enforce this strictly, but I'd say use 120. This looks like 80.

```suggestion
        result.setPearsonCorr(new PearsonsCorrelation().correlation(xValues, yValues)); 
        result.setSpearmanCorr(new SpearmansCorrelation().correlation(xValues, yValues));
```